client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -13,9 +17,12 @@ import NavItem from 'components/section-nav/item';
 import FollowersCount from 'blocks/followers-count';
 import SegmentedControl from 'components/segmented-control';
 import QueryJetpackPlugins from 'components/data/query-jetpack-plugins';
+import { isPluginActive } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { UNITS as StoreStatsTabs } from 'extensions/woocommerce/app/store-stats/constants';
```

Comment:
Thanks @mcsf 

>I don't think it's a precedent we want to set.

I think you are absolutely right. The import is using the tabs in Store Stats as a filter to make sure a link exists on the Store Stats side
```js
[ 'day', 'week', 'month', 'year' ]
```
There are discussions happening about changing Stats Navigation that will make this check obsolete, https://github.com/Automattic/wp-calypso/issues/17291.

I'll open a separate PR to address the bad pattern in the meantime.